src/System.Security.Cryptography.Xml/tests/EncryptionMethodTests.cs

```diff
@@ -45,7 +45,7 @@ public void KeySize_Set_SetsValue(int value)
         public void KeySize_SetNegativeValue_ThrowsArgumentOutOfRangeException(int value)
         {
             EncryptionMethod method = new EncryptionMethod();
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("value", () => method.KeySize = value);
+            Assert.Throws<ArgumentOutOfRangeException>(() => method.KeySize = value);
```

Comment:
@bartonjs is the message that important? I believe I was seeing some Format exception when used that xunit API but will investigate further tomorrow